recipes/pyeit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyeit" %}
+{% set version = "1.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/liubenyuan/pyEIT/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6eac2f59a53386169bf677927b6177436e1909e916fd60ffe161f27c1a3680db
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - pandas
+    - python >=3.6
+    - scipy
+    - matplotlib-base
+    - shapely
```

Comment:
This looks ok to me, but note that it doesn't match the `setup.cfg` of the source package:
- `pathspec >= 0.5.3` isn't used
- `setuptools` isn't an install requirements (it's a build requirement that should be defined in pyproject.toml. See https://setuptools.pypa.io/en/latest/userguide/quickstart.html)
- `pyeit` shouldn't be self referencing itself. You shouldn't import it in the `setup.py`.

I'd recommend you try to fix that in next release.
Again this isn't an issue with the conda recipe but with the source package.

Nothing blocking for this PR.

Reply:
Thank you, These imports are removed from `setup.cfg` and I force updating the tag `v1.1.5` and the corresponding `sha256` in `meta.yaml`.

Reply:
Overwriting a tag is really not recommended, especially as you pushed that version to pypi...
This package https://pypi.org/project/pyeit/1.1.5/ now doesn't match what you have in the git repo.

Always better to bump the version.

I'd suggest to put back the old tag and bump the version to 1.1.6.

Reply:
Ok, version 1.1.5 is reverted to the correct commit, and a new version 1.1.6 is bumped.